ci/ci/templates/batch.html

```diff
@@ -3,18 +3,21 @@
 {% block content %}
     <h1>Batch {{ batch['id'] }}</h1>
     {% if 'attributes' in batch %}
+    <h2>Attributes</h2>
     <div class="attributes">
       {% for name, value in batch['attributes'].items() %}
       <div>{{ name }}: {{ value }}</div>
       {% endfor %}
-      <br>
+    </div>
+    {% endif %}
+    <h2>Timing</h2>
```

Comment:
Sorry to be picky. I think the Timing block should come before the Attributes because it's not optional.

Reply:
No problem, I have no preference so I'll flip them.